scripts/deploy_schemas_to_node.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -m
+
+#Useful for future debugging
+npm --version
+ldd --version
+whoami
+
+/frequency/frequency --dev \
+    -lruntime=debug \
+    --instant-sealing \
+    --wasm-execution=compiled \
+    --execution=wasm \
+    --no-telemetry \
+    --no-prometheus \
+    --port=30333 \
+    --rpc-port=9933 \
+    --ws-port=9944 \
+    --rpc-external \
+    --rpc-cors=all \
+    --ws-external \
+    --rpc-methods=Unsafe \
+    --tmp \
+    &
+
+cd frequency/schemas
+npm install && npm run deploy
```

Comment:
Also non-blocking, but you might find it worthwhile to do the install inside the docker build. While it is a larger image, you don't have to fetch the deps on each start of the image.